### PR TITLE
Side nav animation

### PR DESCRIPTION
**Pull request by zikowang**
| Status  | Type  | Env Vars Change | Review App | Ticket |
| :---: | :---: | :---: | :--: | :--: |
| Ready | Feature | No | [Link](<Review app link here>) | [Link](<ticket link here>) |

## Description

Add animation to side nav.
Adjusted Layout for that.

## Motivation and Context

Good feeling when using the side nav.

## How Has This Been Tested?
locally

## Screenshots / GIFs (if appropriate):

Initial settings screen
image

Switch between side nav variations
Mar-25-2021 13-06-10

Top nav adjustment
Mar-25-2021 15-17-10

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Enhancement (non-breaking change which cleans up / improves existing functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [x] I have read the **CONTRIBUTING** document.
- [ ] I have considered the accessibility of my changes (i.e. did I add proper content descriptions to images, or run my changes with talkback enabled?)
- [x] I have documented my code if needed

### CODE REVIEW

**Comment by luwol03** [comment]
```js
import { useDispatch, useSelector } from "react-redux";
import { setMenuStyle } from "../../redux/Actions/setting";

const MyComponent = () => {
  const dispatch = useDispatch();
  const menuStyle = useSelector((state) => state.setting.menuStyle);

  return (
    <div>
      <p>{menuStyle}<p>
      <Button
          onClick={() => {
            dispatch(setMenuStyle({ menuStyle: menuStyle === "default" ? "fancy" : "default" }));
          }}
        >
    </div>
  )
}
```

File: src/Components/Nav/TopNav.jsx [hunk 1]
```diff
@@ -27,22 +28,26 @@ function TopNav() {
 
   return (
     <div className="top-nav">
+      <div className="title">
+        <h1 className="text">{t("global.vocascan")}</h1>
+      </div>
       <h3 className="user" onClick={handleClick}>
+        <PersonIcon />
```
Comment (noctera) [inline src/Components/Nav/TopNav.jsx:35]: Icon is not in in line with Text
Unbenannt

Comment (zikowang) [reply src/Components/Nav/TopNav.jsx:31]: ich hab noch die top nav vom Verhalten her angepasst.

File: src/Components/Nav/TopNav.jsx [hunk 2]
```diff
@@ -1,22 +1,19 @@
-import React, { useState } from "react";
+import React from "react";
 import { useSelector, useDispatch } from "react-redux";
 import { useTranslation } from "react-i18next";
+import PersonIcon from "@material-ui/icons/Person";
 
 import Button from "@material-ui/core/Button";
 
 import { signOut } from "../../redux/Actions/login";
+import { NavLink } from "react-router-dom";
```
Comment (luwol03) [inline src/Components/Nav/TopNav.jsx:9]: could we move this before the self written components?

**Review by luwol03 — APPROVED** [review]

**Pull request merged**